NAPS2.Lib/NAPS2.Lib.csproj

```diff
@@ -31,14 +31,14 @@
   <ItemGroup>
     <ProjectReference Include="..\NAPS2.Sdk\NAPS2.Sdk.csproj" />
     <ProjectReference Include="..\NAPS2.Escl.Server\NAPS2.Escl.Server.csproj" />
-    <PackageReference Include="Autofac" Version="7.1.0" />
+    <PackageReference Include="Autofac" Version="8.0.0" />
     <PackageReference Include="Ben.Demystifier" Version="0.4.1" />
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
-    <PackageReference Include="MimeKit" Version="1.22.0" />
+    <PackageReference Include="MimeKit" Version="4.4.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="NLog" Version="5.2.6" />
-    <PackageReference Include="Eto.Forms" Version="2.8.2" />
-    <PackageReference Include="NLog.Extensions.Logging" Version="5.3.5" />
+    <PackageReference Include="NLog" Version="5.2.8" />
+    <PackageReference Include="Eto.Forms" Version="2.8.3" />
+    <PackageReference Include="NLog.Extensions.Logging" Version="5.3.8" />
```

Comment:
Anmerkung: Wir updaten mit diesem PR einige Versionen nur im Patch, `Autofac` und `MimeKit` allerdings in der Major-Version. (s. [SemVer](https://semver.org/lang/de/) bzgl. dieser Begriffe)

Diese Änderung könnte zu uns unbekannten Fehlverhalten führen. Sie bringt uns außerdem stärker out-of-sync mit dem Upstream-Projekt, falls dort diese Dependencies noch nicht aktualisiert wurden.

Ich schlage vor, dass diese Versions-Upgrade rückgängig gemacht werden und wir testen, ob unsere Addition noch funktioniert.

Falls diese Upgrades ganz bewusst getan wurden, dann gib mir Bescheid, was ich verpasst habe :)

Reply:
Die Anmerkung gilt für alle Projekt-Dependencies, ich sehe dass es auch andere gibt, wo Major oder Minor-Versionen erhöht wurden.